src/orion/core/cli/checks/presence.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+:mod:`orion.core.cli.checks.presence` -- Presence stage for database checks
+===========================================================================
+
+.. module:: presence
+    :platform: Unix
+    :synopsis: Checks for the creation of a `Database` object.
+
+"""
+
+from orion.core.utils.decorators import register_check
+from orion.core.utils.exceptions import CheckError
+
+
+class _Checks:
+    checks = []
+
+
+class PresenceStage:
```

Comment:
There isn't much to test so far in the configuration. What would be useful is to print the current config after each step so that the user can see how global, env vars and local config file have precedence over the others.